mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/DialogService.kt

```diff
@@ -6,8 +6,8 @@ import no.nav.mulighetsrommet.api.clients.dialog.VeilarbdialogClient
 class DialogService(
     private val veilarbdialogClient: VeilarbdialogClient,
 ) {
-    suspend fun sendMeldingTilDialogen(fnr: String, accessToken: String, dialogRequest: DialogRequest): DialogResponse? {
-        return veilarbdialogClient.sendMeldingTilDialogen(fnr, accessToken, dialogRequest)
+    suspend fun sendMeldingTilDialogen(accessToken: String, dialogRequest: DialogRequest): DialogResponse? {
```

Comment:
Lurer også på om vi har behov for denne klassen i det hele tatt, da det kun er et proxy-kall til klienten her. Kunne kanskje bare kalt direkte på klienten fra routes-filen?

Evt. flyttet noe av logikken til `DelMedBrukerService`, for det er vel egentlig ifm. dette at vi ønsker å gjøre et kall til dialogen. Det er litt merkelig at vi har et endepunkt der man kan sende alle mulige dialogmeldinger :thinking:

(Beklager litt off-topic diskusjon :sweat_smile: )

Reply:
Ja, kunne sikkert kvittet oss med DialogService helt og bare proxyet requesten videre til Dab sitt endepunkt direkte fra routes-filen

Reply:
Jeg gikk for å bare fjerne `dialogService` helt